pkg/txhandler/simple/policyloop.go

```diff
@@ -24,10 +24,10 @@ import (
 	"github.com/hyperledger/firefly-common/pkg/fftypes"
 	"github.com/hyperledger/firefly-common/pkg/i18n"
 	"github.com/hyperledger/firefly-common/pkg/log"
+	"github.com/hyperledger/firefly-transaction-manager/internal/persistence"
```

Comment:
this import is allowed because the example `Simple` transaction handler is in the same repo as the persistence interface.

Reply:
I don't believe that's true, because the `interface` that embeds it has exported it out.

I can certainly update it to redefine the interface.... one sec

Reply:
Done in https://github.com/kaleido-io/firefly-transaction-manager/pull/2/commits/93488e5cac7dc7d87272d488c61c910e50418309

Reply:
Also need to export the SortDirections.

Do we want to do the same for Metrics so that they are in the same coding syle?